src/views/players/view_players.cpp

```diff
@@ -119,25 +119,47 @@ namespace big
 			// used to account for scrollbar width
 			has_scrollbar = window_height + window_pos > (float)*g_pointers->m_gta.m_resolution_y - 10.f;
 
-			// basically whichever is smaller, the max available screenspace or the calculated window_height
-			window_height = has_scrollbar ? (float)*g_pointers->m_gta.m_resolution_y - (window_pos + 40.f) : window_height;
-
 			ImGui::PushStyleColor(ImGuiCol_FrameBg, {0.f, 0.f, 0.f, 0.f});
 			ImGui::PushStyleColor(ImGuiCol_ScrollbarBg, {0.f, 0.f, 0.f, 0.f});
 
-			if (ImGui::BeginListBox("##players", {ImGui::GetWindowSize().x - ImGui::GetStyle().WindowPadding.x * 2, window_height}))
+			auto width_of_player_list   = ImGui::GetWindowSize().x - ImGui::GetStyle().WindowPadding.x * 2;
+
+			static std::string search_player_name;
+			ImGui::SetNextItemWidth(width_of_player_list);
+			if (components::input_text_with_hint("###search_player_name", "search name", search_player_name))
```

Comment:
We no longer allow static text, this must be added to the translation dictionary located here: https://github.com/YimMenu/Translations

Reply:
done